packages/create-docusaurus/templates/shared/docs/tutorial-basics/create-a-blog-post.md

```diff
@@ -31,4 +31,4 @@ Congratulations, you have made your first post!
 Feel free to play around and edit this post as much you like.
 ```
 
-A new blog post is now available at `http://localhost:3000/blog/greetings`.
+A new blog post is now available at [http://localhost:3000/blog/greetings](http://localhost:3000/blog/greetings).
```

Comment:
BTW I think the following is a simpler way to achieve the same:

```
<http://localhost:3000/blog/greetings>
```

Reply:
Autolinks are being removed in MDX v2, so I'll avoid it for now.

Reply:
Good to know. Thank you.